exporter/remotewriteexporter/DESIGN.md

```diff
@@ -0,0 +1,288 @@
+
+
+# **OpenTelemetry Collector Prometheus Remote Write/Cortex Exporter Design**
+
+Authors: @huyan0, @danielbang907
+
+Date: July 30, 2020
+
+## **1. Introduction**
+
+Prometheus can be integrated with remote storage systems that supports its remote write API. Existing remote storage integration support is included in [Cortex](https://cortexmetrics.io/docs/apis/), [influxDB](https://docs.influxdata.com/influxdb/v1.8/supported_protocols/prometheus/), and many [others](https://prometheus.io/docs/operating/integrations/#remote-endpoints-and-storage).
+
+The following diagram shows an example of Prometheus remote write API usage, with Cortex,n open source, horizontally scalable, highly available, multi-tenant, long term storage, as a remote storage backend.
+
+![Cortex Archietecture](https://raw.githubusercontent.com/open-o11y/opentelemetry-collector/design-doc/exporter/cortexexporter/cortex.png)

```

Comment:
Use relative path `(./cortex.png)` same for the other png. May consider to move the pngs in a sub-directory.